lib/spawn-normal.js

```diff
@@ -1,10 +1,15 @@
 
 var spawn = require('child_process').spawn
+var _ = require('lodash')
 
 module.exports = function (command, args, options, done) {
+  // Prevent https://github.com/joyent/node/issues/9158
+  options = _.extend({}, options)
+
+  // Trap any other synchronous errors
   try {
     done(null, spawn(command, args, options))
-  } catch (e) {
-    done(127)
+  } catch (err) {
```

Comment:
Spawn shouldn't ever throw unless there's a bug in node, which of course there is: https://github.com/joyent/node/issues/9158